packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -166,7 +166,7 @@ export class EpochTracker implements IPersistedFileCache {
             await this.checkForEpochError(error, epochFromResponse, fetchType);
             throw error;
         }).catch((error) => {
-            const fluidError = normalizeError(error, {props: {"X-RequestStats": clientCorelationId}});
+            const fluidError = normalizeError(error, { props: { requestStatsHeader: clientCorrelationId}});
```

Comment:
I feel we should either name it how it's named on wire, or given it more clear name. I.e. clientRequestId or something similar. Given that X- is common way to call headers in SPO, the intention of this property was more clear before